lib/rubocop/cop/rspec/rails/minitest_assertions.rb

```diff
@@ -186,8 +119,57 @@ def assertion
             end
           end
 
+          # :nodoc:
+          class InDeltaAssertion < BasicAssertion
+            MATCHERS = %i[
+              assert_in_delta
+              assert_not_in_delta
+              refute_in_delta
+            ].freeze
+
+            NODE_MATCHER_PATTERN = <<~PATTERN
+              (send nil? {:assert_in_delta :assert_not_in_delta :refute_in_delta} $_ $_ $!{sym str}? $_?)
+            PATTERN
+
+            def self.match(expected, actual, delta, failure_message)
+              return nil if delta.empty? && !failure_message.empty?
```

Comment:
Do you think it is possible to pass a message without a delta to this assertion? I have doubts by just looking at the signature

Reply:
No, I wondered that myself but I don't see how it's possible since it doesn't use keywords

Reply:
Should we remove this guard statement then?

Reply:
No this is needed because Rubocop ends up matching even if delta isn't present - I think it's a combo of how optional captures and specific captures work; I think it does something like "this doesn't match `${sym str}?` but it does match `$_` after that".

i.e. this test will break without this guard: https://github.com/rubocop/rubocop-rspec/pull/1791/files#diff-48b7271a8a66b1c5d7538b2fd3dbdb450bf9b78a6852855ddfbcb796260bdc2dR388-R392